networkx/algorithms/centrality/domirank.py

```diff
@@ -0,0 +1,305 @@
+"""DomiRank Centrality: By Marcus Engsig (@mengsig)"""
+
+import networkx as nx
+from networkx.utils import not_implemented_for
+
+__all__ = ["domirank"]
+
+
+@not_implemented_for("multigraph")
+@nx._dispatchable(edge_attrs="weight")
+def domirank(
+    G, analytical=False, sigma=0.95, dt=0.1, epsilon=1e-5, max_iter=1000, check_step=10
+):
+    r"""Compute the DomiRank centrality for the graph G.
+
+    DomiRank centrality computes the centrality for a node by adding
+    1 minus the centrality of its neighborhood. This essentially finds the
+    dominance of a node in its neighborhood, where the parameter $\sigma$ determines
+    the amount of competition in the system. The competition parameter $\sigma$
+    tunes the balance of DomiRanks integration of  local and global topological
+    information, to find nodes that are either locally or globally Dominant. It is
+    important to note that for the iterative formulation of DomiRank (as seen below),
+    the competition parameter is bounded: $\sigma \in [0,\frac{1}{-lambda_N}].
+    DomiRank is defined as the stationary solution to the dynamical system:
+
+    .. math::
+
+        \frac{d \Gamma_i(t)}{dt} = \sigma (d_i - \sum_j A_{ij} \Gamma_j(t)) - \Gamma_i(t),
```

Comment:
Very, very minor nit: the differential operator should be [typeset upright](https://tex.stackexchange.com/questions/14821/whats-the-proper-way-to-typeset-a-differential-operator).